README.md

```diff
@@ -1,21 +1,50 @@
-# styleguide
-Mapzen Styleguide powering everything part of the Mapzen brand
+Styleguide
+==========
 
-Checkout the [wiki](https://github.com/mapzen/styleguide/wiki)
+Mapzen Styleguide makes it easy for us to create right-looking websites that
+work well across platforms and into the future. If you’re publishing Mapzen
+stuff, the guide is for you.
 
-#### Clone and install
+🔋 Batteries included:
 
-```sh
-git clone git@github.com:mapzen/styleguide.git
-npm install
-```
+* [Basic page structure](https://mapzen.com/common/styleguide/index.html) with
+  standard linked CSS and Javascript resources.
+* [Sample UI components](https://mapzen.com/common/styleguide/ui-components.html)
+  like navigation, social buttons, search boxes, and pagination that can be
+  combined to make new kinds of pages.
+* [Complete sample pages](https://mapzen.com/common/styleguide/#sample-pages)
+  like blog posts and developer documentation showing how all the pieces fit
+  together.
 
-#### Run locally
+Use
+---
 
-```sh
-gulp build
-npm start
-open http://localhost:3000/
-```
+To use the guide for Mapzen-looking web sites, visit
+[https://mapzen.com/common/styleguide](https://mapzen.com/common/styleguide)
+and follow the instructions to structure HTML and linked style and script
+resources.
 
-To watch files for changes, run `gulp` in a separate terminal instance.
+Contribute
+----------
+
+Watch and contribute to development on the styleguide via issues
+[here in Github](https://github.com/mapzen/styleguide/issues)
+and [on Waffle.io](https://waffle.io/mapzen/styleguide).
+
+To edit and test Styleguide, please work on a new branch in this repository and
+[use Precog](http://precog.mapzen.com/mapzen/styleguide) for preview and
+testing.
+
+🚧 `master` branch is live 🚧
+
+If you’d like to edit locally, Styleguide requires [Node](https://nodejs.org/)
+and can be built and tested with these commands:
+
+1. `npm install`
+2. `gulp build` (or `./node_modules/.bin/gulp build` if _gulp_ not found)
```

Comment:
how about we adding little bit more details about what Gulp does? (aggregating file inside of src/site folder and spit into dist folder, that kind of stuffs in case anybody wants to tweak and see)